lgc/lgc.go

```diff
@@ -306,13 +306,23 @@ func (lgd *LoadGeneratingConnectionDownload) doDownload(ctx context.Context) {
 		return
 	}
 
+	// Used to disable compression
+	request.Header.Set("Accept-Encoding", "identity")
+
 	lgd.downloadStartTime = time.Now()
 	lgd.lastIntervalEnd = 0
 
 	if get, err = lgd.client.Do(request); err != nil {
 		lgd.valid = false
 		return
 	}
+
+	// Header.Get returns "" when not set
+	if get.Header.Get("Content-Encoding") != "" {
+		lgd.valid = false
+		fmt.Printf("Content-Encoding header was set (compression not allowed)")
+		return
+	}
```

Comment:
Just to confirm: Would it be okay for the server to respond with "identity" listed?

Reply:
Yes; that should also be added as acceptable

Reply:
My mistake on this -- I was looking at the information about `Accept-Encoding`. I do not believe that we need to allow for the possibility of `indentity` in this field. In particular, 

> If one or more encodings have been applied to a representation, the sender that applied the encodings MUST generate a Content-Encoding header field that lists the content codings in the order in which they were applied. Note that the coding named "identity" is reserved for its special role in [Accept-Encoding](https://httpwg.org/specs/rfc9110.html#field.accept-encoding) and thus SHOULD NOT be included.

Sorry!